src/ILVerify/src/ILImporter.Verify.cs

```diff
@@ -949,7 +957,34 @@ void ImportSwitchJump(int jmpBase, int[] jmpDelta, BasicBlock fallthrough)
 
         void ImportBranch(ILOpcode opcode, BasicBlock target, BasicBlock fallthrough)
         {
-            // TODO: Verification rules
+            switch (opcode)
+            {
+                case ILOpcode.br: break;
+                case ILOpcode.brfalse:
+                case ILOpcode.brtrue:
+                    {
+                        StackValue value = Pop();
```

Comment:
Nit: It would be nice to be consistent about var vs. explicit types within a method.